app.js

```diff
@@ -77,7 +78,7 @@ app.use(function(req, res, next) {
 const prefix = "v1"
 app.use("/", index)
 app.use(`/${prefix}/` + `health-check`, healthCheck)
-app.use(`/${prefix}/` + `address`, address)
+app.use(`/${prefix}/` + `address`, address.router)
```

Comment:
Why are you adding `.router` here?

Reply:
You can see in [lines 331-340 of address.js](https://github.com/Bitcoin-com/rest.bitcoin.com/blob/ct-tests/routes/address.js#L331-L340) that I changed the export. It now exports an object with a `router` property and an additional `testableComponents` property that contains the internal functions to be  tested.